### ISSUE TITLE
aktuálnosť Fit for 55

### ISSUE THREAD

**Issue by LenkaSedlackova** [faktaoklimatu/web-cz#1211]
ahoj,

v texte o Fit for 55 (zrejme z konca ledna 2022 podľa verzie infografiky) je odstavec:
Balíček se **aktuálně** nachází ve fázi legislativních návrhů ze strany Evropské komise, které se dále projednávají v Evropském parlamentu a Radě Evropské unie, tedy s účastí ministrů vlád členských zemí a národních pracovních skupin. Pro Českou republiku je v tomto kontextu zásadní předsednictví Radě EU,[2] které **převezme** v druhé polovině roku 2022 a **bude tak mít možnost** ovlivnit směr jednání. Očekává se, že by vyjednávání o finální podobě legislativy mohla trvat až do roku 2024.[3]
vieme aktualizovať? (aj kvôli vznikajúcej SVK verzii)

**Comment by katkolouchova** [comment]
Ano, mám to v backlogu, už jsem to řešila i s Vierkou – zkusím se k tomu brzy dostat :)
